admin_manual/enterprise/external_storage/windows-network-drive_configuration.rst

```diff
@@ -226,7 +226,7 @@ is updated and timing::
   updated storage wnd::administrador@server/share// from mount id 3 -> removed internal path : Capirotes/New Document.txt
   found 1 related storages from mount id 2
 
-See :doc:`../configuration_server/occ_command` for detailed help with ``occ``.
+See :doc:`../../configuration/server/occ_command` for detailed help with ``occ``.
```

Comment:
Hmmm - I guess this relative link was broken before this change, thus the need for adding the ".."
(maybe a few more of those sort of fixes further down)